cron.yaml

```diff
@@ -31,4 +31,7 @@ cron:
   schedule: every 1 minutes
 - description: Admin Directory API logging
   url: /tasks/create_and_log_reports
-  schedule: every 24 hours
\ No newline at end of file
+  schedule: every 24 hours
+- description: log ACL changes on buckets
+  url: /tasks/log_acls
+  schedule: every 1 minutes
```

Comment:
Yes, although I am trying to push another change (I forgot to add request and HttpResponse to views.log_acls) and am having trouble pushing.